selfhost/parser.jakt

```diff
@@ -1458,6 +1461,18 @@ struct Parser {
                     current_param_requires_label = true
                     current_param_is_mutable = false
                     parameter_complete = false
+
+                    // This would allow syntax like
+                    // function a(
+                    //  a: i64
+                    //
+                    //  b: i64
+                    //)
+                    // Do we want to allow that?
```

Comment:
yes, and please test it too :)
also, pls don't put this sort of code comment in the source kthx

Reply:
Removed them

Have already added tests here:

https://github.com/feniljain/jakt/blob/7e1c263d98388b9650d70be2bf36a2b19efb919e/samples/functions/argument_separation.jakt#L13-L17

and here

https://github.com/feniljain/jakt/blob/7e1c263d98388b9650d70be2bf36a2b19efb919e/samples/functions/parameters_separation_comma_newline.jakt#L15-L21